scipy/signal/_short_time_fft.py

```diff
@@ -1130,6 +1133,12 @@ def spectrogram(self, x: np.ndarray, y: np.ndarray | None = None,
         `stft_detrend`, hence all parameters are discussed there. If `y` is not
         ``None`` it needs to have the same shape as `x`.
 
+        The cross-spectrogram may be interpreted as the time-frequency analogon of the
+        cross-spectral density (consult `csd`). If the STFT is parametrized to be a
```

Comment:
You could also briefly state the [coherency spectrum](https://en.wikipedia.org/wiki/Cross-spectrum#:~:text=Squared%20coherency%20spectrum%5Bedit%5D), another useful spectrogram that can be obtained using this functionality.